common/test/com/commercetools/sunrise/common/template/engine/handlebars/PlayJavaFormResolverTest.java

```diff
@@ -0,0 +1,52 @@
+package com.commercetools.sunrise.common.template.engine.handlebars;
+
+import com.commercetools.sunrise.common.forms.ErrorBean;
+import com.commercetools.sunrise.common.forms.ErrorsBean;
+import com.commercetools.sunrise.common.utils.ErrorFormatter;
+import com.google.common.collect.Maps;
+import org.junit.Test;
+import org.mockito.Mockito;
+import play.data.Form;
+import play.data.validation.ValidationError;
+
+import java.util.*;
+import java.util.function.Predicate;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PlayJavaFormResolverTest {
+
+    PlayJavaFormResolver formResolver = new PlayJavaFormResolver(singletonList(Locale.ENGLISH), new ErrorFormatter() {
+        @Override
+        public String format(List<Locale> locales, String message) {
+            return message;
+        }
+    });
```

Comment:
Suggestion: the instantiation of ErrorFormatter can be a bit smaller with:
```java
new PlayJavaFormResolver(singletonList(Locale.ENGLISH), (locales, message) -> message);
```

Reply:
694feaf